Jenkinsfile

```diff
@@ -35,24 +80,33 @@ pipeline {
     //     }
     //   }
     // }
-    stage("Tests") {
+    stage('Build Examples') {
       steps {
-        dir("${REPO}/tests"){
+        dir("${REPO}/examples") {
           viewEnv(){
             withVenv() {
```

Comment:
viewEnv is for the old xgit sandbox, so you should be able to remove all of these - especially as you're getting XTC from withTools.